.gitignore

```
@@ -160,3 +160,5 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
```

```suggestion
*.html
```
vantar að hunsa allar html skrár sem koma með `--debug` flagginu í lið 4